src/app/dataset-view/additional-components/dataset-settings-component/tabs/webhooks/components/list-webhooks/list-webhooks.component.ts

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { ChangeDetectionStrategy, Component, inject, Input, OnInit } from "@angular/core";
+import { AsyncPipe, NgClass, NgIf } from "@angular/common";
+import { BehaviorSubject, Observable, take } from "rxjs";
+import {
+    WebhookSubscriptionStatus,
+    WebhookSubscription,
+    DatasetBasicsFragment,
+} from "src/app/api/kamu.graphql.interface";
+import { ModalService } from "src/app/common/components/modal/modal.service";
+import { promiseWithCatch } from "src/app/common/helpers/app.helpers";
+import RoutingResolvers from "src/app/common/resolvers/routing-resolvers";
+import { DatasetSettingsWebhookTabData } from "../../dataset-settings-webhooks-tab.component.types";
+import { DatasetWebhooksService } from "../../service/dataset-webhooks.service";
+import { WebhooksHelpers } from "../../webhooks.helpers";
+import { MatDividerModule } from "@angular/material/divider";
+import { MatIconModule } from "@angular/material/icon";
+import { MatTableModule } from "@angular/material/table";
+import { FeatureFlagDirective } from "src/app/common/directives/feature-flag.directive";
+import { NavigationService } from "src/app/services/navigation.service";
+import ProjectLinks from "src/app/project-links";
+
+@Component({
+    selector: "app-list-webhooks",
+    standalone: true,
+    imports: [
+        //-----//
+        AsyncPipe,
+        NgClass,
+        NgIf,
+
+        //-----//
+        MatDividerModule,
+        MatIconModule,
+        MatTableModule,
+
+        //-----//
+        FeatureFlagDirective,
+    ],
+    templateUrl: "./list-webhooks.component.html",
+    styleUrls: ["./list-webhooks.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ListWebhooksComponent implements OnInit {
+    @Input(RoutingResolvers.DATASET_SETTINGS_WEBHOOKS_KEY) public webhooksViewData: DatasetSettingsWebhookTabData;
+    public readonly DISPLAY_COLUMNS: string[] = ["event", "status", "actions"];
+    public readonly WebhookSubscriptionStatus: typeof WebhookSubscriptionStatus = WebhookSubscriptionStatus;
+    private readonly _rowsSubject$ = new BehaviorSubject<WebhookSubscription[]>([]);
+    public readonly rows$ = this._rowsSubject$.asObservable();
+    public readonly dataSource$: Observable<WebhookSubscription[]> = this.rows$;
+    private datasetWebhooksService = inject(DatasetWebhooksService);
+    private modalService = inject(ModalService);
+    private navigationService = inject(NavigationService);
```

Comment:
Give it some structure with blank lines.

Reply:
Fixed